dist/alpine/Dockerfile

```diff
@@ -2,16 +2,15 @@ FROM python:3.8-alpine3.12 as k8s_build
 ARG KOPS_VERSION
 ARG HELM_VERSION
 RUN \
-	mkdir /main && \
+	set -x; > /etc/inittab && mkdir /main && \
 	apk update && \
 	# export functions, for download, json parsing, ssh keygen, api keygen
-	apk add --no-cache bash curl jq openssh openssl && \
+	apk add --no-cache bash curl jq openssh openssl util-linux bind-tools && \
```

Comment:
I see bind-tools useful for using `dig` in debugging DNS. What's specifically useful for debugging from `util-linux`?